tests/testthat/test-add_n.tbl_uvregression.R

```diff
@@ -0,0 +1,70 @@
+skip_if_not(is_pkg_installed(c("broom.helpers", "broom", "survival", "survey"), reference_pkg = "gtsummary"))
```

Comment:
Are there any tests that need survival and survey pkgs?

Reply:
i suppose not